source/people/dangelo.rst

```diff
@@ -1,7 +1,10 @@
 Chelsea D'Angelo
 ================
 
-Contact Information
+.. image::  cad_bitmoji_donut.png
```

Comment:
Having some trouble getting the bitmoji image to show up on your personal page for some reason. Could just be a problem with my machine?